### PR TITLE
2019-02-12 updates

### PR DESCRIPTION
**Pull request by SirSpidey**
- object detection 
- small editorial changes

### CODE REVIEW

File: object-detection.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+{:java: .ph data-hd-programlang='java'}
+{:python: .ph data-hd-programlang='python'}
+{:swift: .ph data-hd-programlang='swift'}
+
+<!-- Link definitions -->
+
+[api-ref-v4]: https://{DomainName}/apidocs/visual-recognition-v4
+
+# Custom Object Detection (Beta)
+{: object-detection-overview}
```
Comment (jeffpk62) [inline object-detection.md]: I think this needs a leading `#`?

File: object-detection.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+{{site.data.keyword.visualrecognitionfull}} Custom Object Detection (Beta) identifies items and their location in an image. The service detects these items based on a set of images with labeled training data that you provide.
+{: shortdesc}
+
+Custom Object Detection is a private beta feature and you must have permission from {{site.data.keyword.IBM_notm}} to make calls to the model. [Request access ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://datasciencex.typeform.com/to/c70Ak5){: new_window}. For more information about beta features, see the [Release notes](/docs/services/visual-recognition/release-notes.html#beta).
+{: important}
+
+You train the object detection model to recognize objects that are important to your workflow or domain. For example, detect damage to cars or find machines that need maintenance or perform visual inspections in the field. You can also use object detection to count objects or manage inventory.
```
Comment (jeffpk62) [inline object-detection.md]: Maybe use a comma for the serial items in the "For example" sentence?

File: object-detection.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+![Classification response image](images/cookies-tag.png "An image to show classification")
+
+### Object detection
+{: #obj-detect-detect}
+
+Custom Object detection is similar to custom classification, but the service identifies the location of items in the image. Like classification, the response also includes the label of each item it detected and the confidence in its identification.
+
+The items that are detected are based on the information that you provide when you train an object detection model.
+
+In the following image, the **Analyze images** method of Custom Object Detection, identifies the location of cookies in the image. Each detected object includes the label (in this case, `Cookie`) with its location and a confidence score.
```
Comment (jeffpk62) [inline object-detection.md]: Should the second comma in the first sentence be removed?

File: object-detection.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+![Object detection response image](images/cookies-bbox.png "An image to show object detection")
+
+## How to use Custom Object Detection
+{: #object-detection-sequence}
+
+To use {{site.data.keyword.visualrecognitionshort}} Custom Object Detection, you follow this sequence of steps to set up a custom object detection model:
+
+1.  Create a collection: A collection is a container for your images and training data. See [Create a collection ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/apidocs/visual-recognition-v4#create-a-collection){: new_window} in the v4 API reference.
+1.  Add your images to the collection. You can add single images by URL or by file, or you can upload a .zip file of images. See [Add images ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/apidocs/visual-recognition-v4#add-images){: new_window} in the v4 API reference.
```
Comment (jeffpk62) [inline object-detection.md]: Should `.zip` be formatted somehow?  I use bold for file names and extensions, which probably violates some style guideline. :-)  (This comment applies to a later occurrence, as well.)

File: object-detection.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    "object": "BurntCookie",
+    "location": {
+      "left": 33,
+      "top": 8,
+      "width": 163,
+      "height": 119
+    }
+  }]
+}
+```
```
Comment (jeffpk62) [inline object-detection.md]: Should this have `{: codeblock}` after the example?  (This comment also applies to later example blocks.)

File: recognize-text.md [hunk 6]
```diff
@@ -54,7 +54,7 @@ Text in images is better recognized when it adheres to these guidelines:
 - The Text model is trained mainly on English language words. Text in other languages likely won't be recognized.
```
Comment (jeffpk62) [inline recognize-text.md]: Maybe say "is unlikely to be recognized" to avoid future tense?

File: object-detection.md (hunk 4 above)
Comment (SirSpidey) [reply object-detection.md]: Well, you learn something new every day. Will "codify" it.

**Pull request merged**